### PR TITLE
expression: conv is compatible with mysql when empty string(#34994)

### LINKED ISSUE #34994: conv function has different result from mysql
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)
```
select conv("",2,16);
```

### 2. What did you expect to see? (Required)
Both 5.7 and 8.0
```
MySQL [test]> select conv("",2,16);
+---------------+
| conv("",2,16) |
+---------------+
| NULL          |
+---------------+
1 row in set (0.000 sec)
```

### 3. What did you see instead (Required)
```
[tidb]> select conv("",2,16);
+---------------+
| conv("",2,16) |
+---------------+
| 0             |
+---------------+
1 row in set (0.001 sec)
```

### PR DESCRIPTION
**Pull request by keeplearning20221**
### What problem does this PR solve?

Issue Number: close #34994

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: expression/integration_serial_test.go [hunk 1]
```diff
@@ -1623,6 +1623,9 @@ func TestMathBuiltin(t *testing.T) {
 	result.Check(testkit.Rows("10"))
 	result = tk.MustQuery("SELECT CONV(0b10, 16, 8)")
 	result.Check(testkit.Rows("2"))
+	//conv is compatible with mysql when empty string(#34994)
+	result = tk.MustQuery("SELECT CONV('', 16, 8)")
```
Comment (hawkingrei) [inline expression/integration_serial_test.go:1627]: You can put this test case into a line.

Comment (keeplearning20221) [reply expression/integration_serial_test.go:1627]: ok

**Comment by wshwsh12** [comment]
Please also check the behavior of tikv and tiflash

**Pull request closed without merging**